compaction/compaction.cc

```diff
@@ -564,7 +564,9 @@ class compaction {
 
     virtual compaction_completion_desc
     get_compaction_completion_desc(std::vector<shared_sstable> input_sstables, std::vector<shared_sstable> output_sstables) {
-        return compaction_completion_desc{std::move(input_sstables), std::move(output_sstables)};
```

Comment:
Please keep all the relevant information in the patch changelog. Try `git log` to see how other commits do that.